docs/sources/datasources/azure-monitor/query-editor/index.md

```diff
@@ -44,7 +44,7 @@ In contrast, Azure Monitor Logs can store a variety of data types, each with the
 
 1. In a Grafana panel, select the **Azure Monitor** data source.
 1. Select the **Metrics** service.
-1. Select a resource from which to metrics by using the subscription, resource group, resource type, and resource fields.
+1. Select a resource from which to metrics by using the subscription, resource group, resource type, and resource fields. Multiple resources can also ben selected as long as they belong to the same subscription, region and resource type. Note that only a limited amount of resource types support this feature.
```

Comment:
```suggestion
1. Select a resource from which to query metrics by using the subscription, resource group, resource type, and resource fields. Multiple resources can also be selected as long as they belong to the same subscription, region and resource type. Note that only a limited amount of resource types support this feature.
```

Reply:
Maybe we should reference the list of supported resource types?

Reply:
There is no good point to reference, pointing code seems too verbose and there is no docs related to this (I mean Azure docs).